en/core-api/direct-debit.md

```diff
@@ -170,7 +179,28 @@ You will get the API response like the following.
 }
 ```
 
+**Response Body**
+
+| Element            | Description                                                  | Type   | Notes                                                        |
+| ------------------ | ------------------------------------------------------------ | ------ | ------------------------------------------------------------ |
+| status_code        | This is the status of the API call.                          | String | For more details, refer to [Status Codes and Error](/en/technical-reference/error-response-code.md#status-codes-and-errors). |
```

Comment:
We can remove "This is the" to make it consistent with the rest of the table entries. Replace with " The status of the API Call". Make this change in all methods and in all files.